.github/ISSUE_TEMPLATE/icon_request.yml

```
@@ -0,0 +1,48 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. Before opening a new issue, please
+        search for duplicate or closed issues or PRs. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction to the original
+        post.
+
+  - type: input
+    attributes:
+      label: Brand Name
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
```

I think we should add a description here to cover (at least) when the request is for a non-web brand, something like:

```suggestion
    attributes:
      label: Website
      description: >-
        For non-web brands you can add a relevant link or simply "None" if you don't think there's a website.
```